README.md

```diff
@@ -1,8 +1,5 @@
 # Test Bayes
 
-[![Build Status](https://travis-ci.org/Test-Bayes/test-bayes.svg?branch=master)](https://travis-ci.org/Test-Bayes/test-bayes/)
```

Comment:
We need to keep the BuildStatus one. Just remove line 4 which is the coverage status

Reply:
Resolved